<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SNPSelectionDialog</class>
 <widget class="QDialog" name="SNPSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select region</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Variant Selection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radiob_cview">
     <property name="text">
      <string>Current View</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radiob_chrom">
     <property name="text">
      <string>Chromosome</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget_chroms">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="radiob_custom">
     <property name="text">
      <string>Custom</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit_selectedsnps">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="plainText">
      <string>Replace this text with custom SNP input (without quote)

For currently selected chromosome : 
&quot;255-345, 541, 566, 577-890&quot;
	or 
&quot;CHROM1:25-900
CHROM2:0-450 (Position numbering starts from 0)&quot;
	or
&quot;25-900
CHROM2:0-300&quot;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="radioButton_bed">
     <property name="text">
      <string>BED File</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="lineEdit_bedfile"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_noncoding">
        <property name="text">
         <string>Non-coding</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_depth">
          <property name="text">
           <string>Depth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_depth">
          <property name="text">
           <string>val</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_qual">
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_quality">
          <property name="text">
           <string>val</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_indels">
        <property name="text">
         <string>Indels</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>167</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton_save">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>166</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
